subsys/bluetooth/mesh/Kconfig

```diff
@@ -52,6 +52,7 @@ config BT_MESH_NLC_PERF_CONF
 	bool "Enable NLC performance configurations"
 	select BT_MESH_V1d1
 	select BT_MESH_COMP_PAGE_2
+	select BT_MESH_COMP_PAGE_1
```

Comment:
```
warning: BT_MESH_COMP_PAGE_1 (defined at subsys/bluetooth/host/../mesh/Kconfig:1388, subsys/bluetooth/host/../mesh/Kconfig:1388) has direct dependencies (BT_MESH_MODEL_EXTENSIONS && BT_MESH_V1d1 && BT_MESH && BT_HCI_HOST && BT_RPC_STACK) || (BT_MESH_MODEL_EXTENSIONS && BT_MESH_V1d1 && BT_MESH && BT_HCI_HOST && BT_HCI && BT) with value n, but is currently being y-selected by the following symbols:
 - BT_MESH_NLC_PERF_CONF (defined at /home/alkh-local/ncs/nrf/subsys/bluetooth/mesh/Kconfig:51), with value y, direct dependencies BT_MESH && BT (value: y), and select condition BT_MESH && BT (value: y)
```
Seems it needs more depende

Reply:
light_dimmer sample. Didn't check the others.

Reply:
It gets enabled: `with value n, but is currently being y-selected`, but Ok, I will explicitly enable `BT_MESH_MODEL_EXTENSION`.

Reply:
Fixed